examples/helm-template/README.md

```diff
@@ -0,0 +1,36 @@
+# Helm Template
+
+This is an example of invoking the helm template function using declarative configuration.
+
+## Function invocation
+
+The function is invoked by authoring a function configuration in local-configs/example.yaml
+with `metadata.annotations.[config.kubernetes.io/function]` set to
```

Comment:
I'm not a fan of these of sentences:

Some words, with 'field X' in some other words in some 'file Foo'.

Might as well just show the YAML file instead of translating it into prose (which is less precise and more confusing).

Reply:
Removed, hope it's less confusing.